recipes/genodsp/build.sh

```
@@ -0,0 +1,9 @@
+#!/bin/bash -e
+set -uex
+
+mkdir -vp ${PREFIX}/bin
+
+make VERBOSE=1 -j ${CPU_COUNT}
+
+cp "$SRC_DIR/genodsp" "$PREFIX/bin"
+chmod ua+x "$PREFIX/bin/genodsp
```

Shellcheck

[error] Couldn't parse this double quoted string. Fix to allow more checks.

(SC1073)